examples/syn.rs

```diff
@@ -0,0 +1,44 @@
+// Example provided in #4
+// https://github.com/jmg-duarte/sealed-rs/issues/4
+
+use proc_macro2::*;
+use sealed::sealed;
+use syn::spanned::Spanned;
+
+#[sealed]
+pub trait AsSpan {
+    fn as_span(&self) -> Span;
+}
+// expands to:
+// pub trait AsSpan: private::Sealed {
+//     fn as_span(&self) -> Span;
+// }
+// mod private {
```

Comment:
Here the and below the module name should be corrected to reflect the true naming.